### DISCUSSION TITLE
Deepin Screen Recorder configured by default to record in .gif

### DISCUSSION THREAD

**Discussion by igatjens** [linuxdeepin/developer-center#2984] [Features Request & Ideas | 特性请求 & 头脑风暴]
When Deepin Screen Recorder is newly installed, by default it records in the .gif format, which is of much lower quality than the .mp4 format.

The first time I record after installing Deepin it is quite unpleasant to see that the recording was of poor quality because it was recorded in .gif, I always forget to change it to MP4 the first time.

Furthermore, when new users record their first screenshot and obtain a poor quality .gif, they will think that Deepin Screen Recorder is a poor quality tool, when in fact it is very good, only that it is configured by default in the wrong way.

I use Deepin 20 beta and the version of Deepin Screen Recorder is deepin-screen-recorder 5.8.0.11-1.

**Comment by igatjens** [comment]
This problem still continues in Screen Recorder version 5.8.0.40-1.

![20200920_085513](https://user-images.githubusercontent.com/66392244/93714373-74c75580-fb1f-11ea-884b-8f9bd2fc2b97.jpg)

**Comment by meiyixiang** [comment]
I have recorded and fed back this question and I will make sure why do I save the screen recording as GIF by default

**Comment by meiyixiang** [comment]
For this question we will change the default to MP4 format around 20.6 phase
